tools/Makefile

```diff
@@ -124,7 +124,7 @@ sctk/.compiled: sctk
 
 # The GitHub archive unpacks into SCTK-{40-character-long-hash}/
 sctk: sctk-$(SCTK_GITHASH).tar.gz
-	tar --exclude '*NONE*html' -xmaf sctk-$(SCTK_GITHASH).tar.gz
+	tar zxvf sctk-$(SCTK_GITHASH).tar.gz
```

Comment:
You don't want to restore the 'exclude' thing?  I think it causes some kind of crash on Windows if that's not done.

Reply:
because zxvf is not with `--exclude '*NONE*html' -xmaf`, `--exclude` only don't unzip html file, it idoes not affect the window system. I test it successfullly.